TESTING.md

```diff
@@ -318,6 +319,56 @@ If JS expression, following `@ts-test-if:` evaluates to truthy value, `@ts-expec
 inserted in its place during the generation. All parameters from the test matrix can be used within that
 expression.
 
+## Memory tests
+
+This suite tests client for memory leaks. It works by repeatedly running test code in a loop
+and monitoring V8 heap usage after every iteration. If it then detects that memory usage growth
+with a rate, higher then threshold it will report a memory leak and fail the test.
```

Comment:
```suggestion
and monitoring V8 heap usage after every iteration. If it detects that memory usage grows
with a rate higher than the threshold, it will report a memory leak and fail the test.
```